plugins/periskop/api-report.md

```diff
@@ -0,0 +1,59 @@
+## API Report File for "@backstage/plugin-periskop"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { ApiRef } from '@backstage/core-plugin-api';
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { ConfigApi } from '@backstage/core-plugin-api';
+import { DiscoveryApi } from '@backstage/core-plugin-api';
+import { Entity } from '@backstage/catalog-model';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// @public
+export const isPeriskopAvailable: (entity: Entity) => boolean;
+
+// @public
+export const PERISKOP_NAME_ANNOTATION = 'periskop.io/name';
+
+// @public
+export class PeriskopApi {
+  // Warning: (ae-forgotten-export) The symbol "Options" needs to be exported by the entry point index.d.ts
+  constructor(options: Options);
```

Comment:
might make this PeriskopApiOptions and export it

Reply:
some other missing exports below too

Reply:
Done, thanks.